src/main/java/io/phdata/streamliner/schemadefiner/configbuilder/ScriptCommand.java

```diff
@@ -220,7 +238,111 @@ private static void build(
     }
   }
 
-  private static TableDefinition getOriginalTable(
+  private static void generateDeltaChangeSummary(
+      ConfigurationDiff configDiff,
+      Configuration config,
+      Map<String, Map<String, String>> typeMapping,
+      String outputDirectory) {
+    // this does not have deleted tables. deleted tables are already removed from this list.
+    List<TableDiff> tableDiffs = configDiff.getTableDiffs();
+    Set<SchemaChanges> validSchemaChanges = ((Jdbc) config.getSource()).getValidSchemaChanges();
```

Comment:
> (Jdbc) config.getSource()

Feels like we should check if this is true and return void if not as opposed to throwing class cast exception

Reply:
updated